### PR TITLE
docs: add details about styles in the smart CDN documentation

### LINKED ISSUE #523: docs: update docs for smart CDN (add styles)

### PR DESCRIPTION
**Pull request by hsablonniere**
Fixes #523

### CODE REVIEW

File: docs/guides/how-to-load-components-via-our-smart-cdn.md [hunk 1]
```diff
@@ -3,21 +3,39 @@ kind: '📌 Docs'
...
+
+First, you need a `<link>` tag to load the styles for the default theme:
+
+```html
+<link rel="stylesheet" href="https://components.clever-cloud.com/styles.css?version=9.0.0">
+```
+
+INFO: This URL supports one query param: `version`.
+
+WARNING: This `<link>` tag is only need starting from version `9.0.0`.
```
Comment (florian-sanders-cc) [inline docs/guides/how-to-load-components-via-our-smart-cdn.md:17]: needed* or required*

File: docs/guides/how-to-load-components-via-our-smart-cdn.md [hunk 2]
```diff
@@ -3,21 +3,39 @@ kind: '📌 Docs'
...
+<script type="module" src="https://components.clever-cloud.com/load.js?version=9.0.0&lang=fr&components=cc-toggle,cc-tile-requests"></link>
 ```
 
+INFO: This URL supports three query params: `version`, `lang` and `components`.
+
+We provide a UI where you can select the version, language and list of components and get the right URLs to use:
+
+https://components.clever-cloud.com/
+
+You can refer to the sections below to know more about the different query params.
```
Comment (florian-sanders-cc) [inline docs/guides/how-to-load-components-via-our-smart-cdn.md:31]: learn more*

File: docs/guides/how-to-load-components-via-our-smart-cdn.md [hunk 3]
```diff
@@ -3,21 +3,39 @@ kind: '📌 Docs'
...
+
+INFO: This URL supports one query param: `version`.
+
+WARNING: This `<link>` tag is only required starting from version `9.0.0`.
+
+Then you need a `<script>` tag to load the components and init the i18n system if necessary:
 
 ```html
-<script type="module" src="https://components.clever-cloud.com/load.js?version=5.3.1&lang=fr&components=cc-toggle,cc-tile-requests"></script>
+<script type="module" src="https://components.clever-cloud.com/load.js?version=9.0.0&lang=fr&components=cc-toggle,cc-tile-requests"></link>
```
Comment (roberttran-cc) [inline docs/guides/how-to-load-components-via-our-smart-cdn.md:22]: Accidental copy paste on closing tag?

**Review by roberttran-cc — COMMENTED** [review]
Except the one typo, LGTM!

Edit: woops, didn't see it was already merged. :sweat_smile:

**Pull request merged**